### ISSUE TITLE
Coherency between parameters in pages 284 and 285

### ISSUE THREAD

**Issue by davidgalley** [avehtari/ROS-Examples#79]
Hello,
First a big thank you for the book, can't wait for the follow-up on advanced regressions and hierarchical models. Still very nice to have a great reminder on basics, excellent refresher!

When reading p284 & 285, I find some incoherence between parameters' values, unless I get it wrong. 
Page 284, the `fit_la` model states that `intercept `median is -2.97, median of parameters for `height` and `male` are respectively 0.07 and 1.66. However, the corresponding equations page 285 (`Pr(earnings>0) = ...`) reads -3.97, 0.8 and 1.70 for these parameters.

The same occurs for the `fit_lb` model

- `(Intercept)` 7.98 (p284) => 8.15 (p285)
- `male` 0.77 (p284) => 0.43 (p285)

Kind regards
David

PS: maybe p284 reports the median and p285 the mean, in this case could be stated

**Comment by andrewgelman** [comment]
Thanks.  I looked at the html file:  https://avehtari.github.io/ROS-Examples/Earnings/earnings_compound.html
and the numbers are very slightly different from the book. I assume that comes from randomness in the Stan fit and the fact that I did not set a seed when running these simulations for the book. A lso there is inconsistency because in the book I call these fit_1a and fit_1b and in the html file they are fit_2a and fit_2b.  But in any case it seems like the numbers on page 284 are basically correct, and the numbers on page 285 are a typo.  So I will change the numbers on page 285 to match those on page 284.
